cfme/services/catalogs/catalog_item.py

```diff
@@ -210,11 +210,14 @@ def create(self):
         sel.force_navigate('catalog_item_new',
                            context={'provider_type': self.item_type})
         sel.wait_for_element(basic_info_form.name_text)
+        catalog = self.catalog or "<Unassigned>"
+        dialog = self.dialog or "<No Dialog>"
```

Comment:
I am quite opposed to having string-based relationships, I prefer having objects.